Core/include/Acts/EventData/Measurement.hpp

```diff
@@ -574,7 +576,8 @@ auto makeMeasurement(source_link_t source,
                      indices_t index0, tail_indices_t... tailIndices)
     -> FixedSizeMeasurement<source_link_t, indices_t,
                             1u + sizeof...(tail_indices_t)> {
-  return {std::move(source), {index0, tailIndices...}, params, cov};
+  indices_t indices[1u + sizeof...(tail_indices_t)] = {index0, tailIndices...};
```

Comment:
Fixed size c-style array. Doesn't really matter, but could it be a `std::array`?

Reply:
Yes, it could. The temporary container is now a `std::array`.